src/main/java/org/opensearch/security/OpenSearchSecurityPlugin.java

```diff
@@ -1484,7 +1485,7 @@ public List<Setting<?>> getSettings() {
 
             settings.add(Setting.boolSetting(ConfigConstants.SECURITY_DISABLED, false, Property.NodeScope, Property.Filtered));
 
-            settings.add(Setting.intSetting(ConfigConstants.SECURITY_CACHE_TTL_MINUTES, 60, 0, Property.NodeScope, Property.Filtered));
```

Comment:
we should not remove default value. Instead we can change it to have Property.Dynamic

Reply:
This was moved to SecuritySettings, the default value remains.